docker/vue-app/src/form_editor/components/form_editor_view/FormQuestionDisplay.js

```diff
@@ -111,7 +110,8 @@ export default {
                             :title="hasCode ? 'Open Advanced Options. Advanced options are present.' : 'Open Advanced Options.'">
                             Programmer
                         </button>
-                        <img v-if="hasCode" :src="libsPath + 'dynicons/svg/document-properties.svg'" alt="" title="advanced options are present" />
+                        <img v-if="conditionalQuestion" :src="libsPath + 'dynicons/svg/go-jump.svg'" alt="" title="conditional logic is present" />
+                        <img v-if="hasCode" :src="libsPath + 'dynicons/svg/application-x-executable.svg'" alt="" title="advanced options are present" />

```

Comment:
The conditionalQuestion value could be used for the 'modify logic' button to include in the title whether conditions exist.  Screen-readers will read button titles (but not img titles), so presenting this info up front to screen reader users seemed better than including it in the alt attrs.

Since both images are conditional, a "margin-left:auto" style could be added to the second one inline here.  This should consistently put the diamond on the right side regardless of whether the other one is showing.